crates/sui-benchmark/tests/simtest.rs

```diff
@@ -405,6 +406,26 @@ mod test {
         default_num_validators: usize,
         default_epoch_duration_ms: u64,
     ) -> TestCluster {
+        sui_node::set_jwk_injector(Arc::new(|_authority, provider| {
```

Comment:
Why is this needed by the benchmark?

Reply:
It's just so that the authenticator state code is running during theses tests, especially crash and restart tests

Reply:
Should we put it in the test cluster code instead of here so it has more coverage?